internal/luks2/cryptsetup_test.go

```diff
@@ -180,13 +183,19 @@ func (s *cryptsetupSuite) TestFormatOptionsValidateBadMetadataSize(c *C) {
 }
 
 func (s *cryptsetupSuite) TestFormatOptionsValidateBadKeyslotsAreaSize(c *C) {
+	// minimum size of a keyslot is keysize*4000 in KiB
+	// (250 KiB for 64bit keys, 125 for 32bit keys)
+	minSize := (KeySize(SelectCipher()) * 4000) / 1024
 	for _, opts := range []FormatOptions{
-		{KeyslotsAreaKiBSize: 128},
+		// smaller than the minimum
+		{KeyslotsAreaKiBSize: minSize},
```

Comment:
I was a bit confused about why this works because this is actually just the minimum size, but it passes because it actually fails the "must be a multiple of 4KiB" check.